trafficcloud/pdf_generate.py

```diff
@@ -0,0 +1,27 @@
+import os
+from fpdf import FPDF
+from PIL import Image
+
+def makePdf(pdfFileName, listImages, dir = ''):
+    """
+    Arguments
+    ---------
+    pdfFileName: str, i.e. 'output'
+    listImages: list of str, i.e. ['road_user_type_counts.png', 'speed_cdf.png']
+    dir: str, path where images located, and pdf will be outputted
+    """
+    if (dir):
+        dir += "/"
```

Comment:
Probably do some better checking just in case somebody puts in a string with a '/' already at the end?